device/mellanox/x86_64-nvidia_sn2201-r0/sensors.conf

```diff
@@ -33,15 +90,71 @@ bus "i2c-8" "i2c-1-mux (chan_id 6)"
 # Power controllers
 bus "i2c-9" "i2c-1-mux (chan_id 7)"
     chip "pmbus-i2c-*-40"
-    ignore curr3
-    ignore in3
-    ignore power3
+        label curr1 "VR IC PSU 12V Rail curr(in)"
+        label curr2 "ASIC 0.9V VCORE Rail Curr(out)"
+        label in1 "VR IC PSU 12V Rail"
+        label in2 "VR IC ASIC 0.9V VCORE Rail"
+        label power1 "VR IC PSU 12V Rail pwr(in)"
+        label power2 "ASIC 0.9V VCORE Rail Pwr(out)"
+        ignore curr3
+        ignore in3
+        ignore power3
+        label temp1 "VR IC Temp1"
 
 bus "i2c-2" "i2c-1-mux (chan_id 0)"
     chip "ads1015-i2c-*-49"
-    ignore in1
-    ignore in2
-    ignore in4
-    ignore in5
-    ignore in6
-    ignore in7
+        label in0 "ASIC 1.8V Rail"
+        label in3 "MONITOR MB 12V"
+        compute in3 (7.2)*@, @/(7.2)
+        ignore in1
+        ignore in2
+        ignore in4
+        ignore in5
+        ignore in6
+        ignore in7
+
+bus "i2c-8" "i2c-1-mux (chan_id 6)"
+    chip "powr1014-i2c-8-37"
+        label in0 "MONITOR CPU Board V3P3"
+        label in1 "MONITOR CPU Board VR_VCC_1V15"
+        label in2 "MONITOR CPU Board VR_VNN_1V05"
+        label in3 "MONITOR CPU Board VR_VCCRAM_1V15"
+        label in4 "MONITOR CPU Board VR_VDDQ_1V20"
+        label in5 "MONITOR CPU Board V1P05"
+        label in6 "MONITOR CPU Board P2V5_VPP"
+        label in7 "MONITOR CPU Board P0V6_VTT_DIMM"
+        label in8 "MONITOR CPU Board V1P8"
+        label in9 "MONITOR CPU Board V1P24"
+
+# PSU PMBus sensors
+bus "i2c-3" "i2c-1-mux (chan_id 1)"
+    chip "pmbus-i2c-3-58"
+        label in1 "PSU-1 220V Rail(in)"
+        label in2 "PSU-1 12V Rail(out)"
+        ignore in3
+        label fan1 "PSU-1 fan1"
+        label temp1 "PSU-1 temp1"
```

Comment:
temp1 -> Temp1? And same for other temperature label.

Reply:
fixed